documentation/docs/guides/terminal-integration.md

```diff
@@ -0,0 +1,122 @@
+---
+unlisted: true
+---
+# Terminal Integration
```

Comment:
a todo for future, would be nice to show how to set this up with 
https://starship.rs/
and
https://github.com/ianthehenry/zsh-autoquoter